jax/_src/test_util.py

```diff
@@ -845,6 +845,18 @@ def _CheckAgainstNumpy(self, numpy_reference_op, lax_op, args_maker,
                         atol=atol or tol, rtol=rtol or tol,
                         canonicalize_dtypes=canonicalize_dtypes)
 
+_CPP_JITTER = functools.partial(api._jit, True)
+_CPP_JITTER._name = "cpp"
+_PYTHON_JITTER = functools.partial(api._jit, False)
+_PYTHON_JITTER._name = "python"
+_NOOP_JITTER = lambda x: x
```

Comment:
I think I'd leave this out of the list and add it locally in the test cases that explicitly want to test "without jit"? After all, it isn't a jit implementation.

Reply:
All the parameterised tests use the `NOOP` jitter, so for now I think it should remain. Adding it in locally for every parameterised test would just lead to a lot of code duplication without any benefit, I think. Can always refactor this later if there are any cases where the no-op test case is not used.